.cirrus.yml

```diff
@@ -147,18 +146,19 @@ validate_windows_task:
   <<: *ONLY_PR_AND_MAINTAINED_BRANCHES
   env:
     JDK_VERSION: "17"
-    IDEA_HOME: ""
-    RIDER_HOME: ""
-    RESHARPER_HOME: ""
-    CLION_HOME: ""
+    IDEA_HOME: "/C/Program Files/intellij"
+    RIDER_HOME: "/C/Program Files/rider"
+    RESHARPER_HOME: "/C/Program Files/resharper"
+    CLION_HOME: "/C/Program Files/clion"
   ec2_instance:
     <<: *WINVM_DEFINITION
   <<: *SETUP_GRADLE_CACHE
   # skip unit tests from submodules to allow caching (else cache is too big for Cirrus)
   build_script: |
     source cirrus-env QA
     source .cirrus/use-gradle-wrapper.sh
-    ./gradlew :check
+    mkdir "/C/Program Files/Microsoft/jdk-17.0.12.7-hotspot/Packages"
+    ./gradlew :check -x :its:check
```

Comment:
its:check runs other things such as checking licenses so we should run it, it will skip qas anyway